README.md

```diff
@@ -187,46 +189,63 @@ Please examine the XML docs for full details. However, in simplified form, we ha
+                                                                                                        // and *almost* identical to ".With(m => m.Sounds, new []{"Rarrrgggh!", "Screech!", "Woooooh!"})" (only difference is that the containing array is not shared.)
 
-                    .WithFactory(m => m.FavouriteFood, () => new List<string>())            // All monsters will get their own, distinct (initially empty) List<> object for food.
-                    .WithFactory(m => m.Age, () => rand.Next(4))                            // Age might be 2, 4, 1, 4, 3 ...
-                    .WithBuilder(m => m.HolidayLake, LakeBuilder.Minimal)                   // All monsters will have this.HolidayLake populated with the result of "LakeBuilder.Minimal.Build()".
-                    .WithBuilt(m => m.HomeLake)                                             // All monsters will have this.HomeLake populated with the result of "LakeBuilder.New.Build()", because Lake has a registered Builder (and 'New' is used preferentially)
-                    .WithBuilt(m => m.Egg)                                                  // All monsters will have this.Egg populated with "new Egg()", because no builder has been registered for Eggs.
-                    .WithPostBuildSetup(IncludeMonsterInHomeLake)                           // `this.LakeId`, and `this.HomeLake.Monsters` will be updated to honour `this.HomeLake` ... but only at the END of setup. i.e. honouring any later-defined overrides of `this.HomeLake` if configured.
-                    .WithSetup(m =>                                                         // Runs this arbitrary logic against the monster. (But these values could be overridden by later Steps.)
+                    .WithFactory(m => m.FavouriteFood, () => new List<string>())                        // All monsters will get their own, distinct (initially empty) List<> object for food.
+                    
+                    .WithAddToCollection(m => m.FavouriteFood, "People")                                // All monsters like to eat people, in addition to anything that could have been configured prior to this point.
+                                                                                                        // (Obviously the more natural way to achieve that would be to include it in the previous Factory, but we want to demonstrate this .WithAddToCollection method.)
+
+                    .WithFactory(m => m.Age, () => rand.Next(6))                                        // Age might be 2, 4, 1, 4, 6, 3 ...
+                    
+                    .WithBuilder(m => m.HomeLake, LakeBuilder.Default)                                      // All monsters will have this.HomeLake populated with the result of "LakeBuilder.New.Build()"
```

Comment:
Again, I'm not sure if this method (or `WithNew` actually provide anything over `WithFactory` tbh, but maybe that's just me being a simplicist :)

Reply:
I disagree.
This whole library is about making it easier to do these things. Adding helper short-hands for common versions of things that **_could_** be done other ways seems entirely in-scope.

(Plus, if an individual user doesn't think they're clear at the point of usage they can just ignore them)

Reply:
The comment should read: `LakeBuilder.New.Build()`